### DISCUSSION TITLE
NOS3 Unrealistic Telemetry Data Issue

### DISCUSSION THREAD

**Discussion by gracekinney** [nasa/nos3#336] [Q&A]
Hi everyone,

I am working on a project using NOS3 as the main focus for generating different satellite scenarios, where we will use this data for testing. My team and I have been able to get NOS3 running from the VM to an API, and we have successfully set-up the simulation so that the terminal commands make prep, and commands from Igniter: make clean, make, and make launch all work, and we are able to run an instance of the simulation. 

However, we are running into an issue. We are not seeing realistic telemetry data from the simulator when we run different scenario commands. For example, running from the Command Sender: target GENERIC_EPS, command GENERIC_EPS_SWITCH_CC. We have found that when we use Command Sender to turn all switches ON, the BATT_VOLTAGE decreases, and when we turn all switches OFF the BATT_VOLTAGE decreases. We expect to see the opposite: switches ON BATT_VOLTAGE increases, switches OFF BATT_VOLTAGE decreases. We have theorized that the switches may be backwards? 

Please refer to the pdf report I have attached, which highlights unrealistic data and includes screenshots from Telemetry Grapher and Packet Viewer.

The bottom line is that we are experiencing issues with the telemetry data not being realistic, and we believe NOS3 is much more capable than what we are experiencing. 

I would like to ask the developers to help resolve the issue of unrealistic telemetry data, and potentially clarify how and why we are experiencing this.

Thank you!

[SatSim_EPS_demo_report_GK.pdf](https://github.com/user-attachments/files/16266534/SatSim_EPS_demo_report_GK.pdf)

**Comment by jlucas9** [comment]
Hi @gracekinney,
I'm guessing the default setup of the EPS currently in the open source has large current draws and minimal solar array configured for testing purposes at that time. These values ought to be all configurable via the [nos3-simulator.xml](https://github.com/nasa/nos3/blob/main/cfg/sims/nos3-simulator.xml). We just started a new NOS3 effort to improve the default mission captured and expand everything to include tutorials and scenarios for use. Hopefully that will resolve some of the issues experienced here!
